templates/services/nextcloud/nextcloud_yml.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+#create nextcloud.yml
+
+mkdir -p /srv/nextcloud
+
+{
+ echo "services:"
+ echo "  nextcloud:"
+ echo "    image: treehouses/nextcloud"
```

Comment:
The image we want to pull isn't `treehouses/nextcloud`.
It's just `nextcloud`.

You can check the image name by going to https://hub.docker.com/ and searching for the image.
https://hub.docker.com/_/nextcloud